ZenPacks/zenoss/LinuxMonitor/zenpack.yaml

```diff
@@ -200,6 +250,73 @@ device_classes: !ZenPackSpec
                                 dpName: volume_free
                                 lineType: AREA
                                 rpn: "size,-,size,/,-100,*"
+                    Operation Throughput:
+                        units: operations/sec
+                        miny: 0
+                        maxy: 100
+                        graphpoints:
+                            Reads:
+                                dpName: diskstats_readsCompleted
+                                lineType: LINE
+                            Writes:
+                                dpName: diskstats_writesCompleted
+                                lineType: LINE
+                    Merge Rate:
+                        units: operations merged/sec
+                        miny: 0
+                        maxy: 100
+                        graphpoints:
+                            Reads:
+                                dpName: diskstats_readsMerged
+                                lineType: LINE
+                            Writes:
+                                dpName: diskstats_writesMerged
+                                lineType: LINE
+                    Sector Throughput:
+                        units: sectors/sec
+                        miny: 0
+                        maxy: 100
+                        graphpoints:
+                            Read:
+                                dpName: diskstats_sectorsRead
+                                lineType: LINE
+                            Written:
+                                dpName: diskstats_sectorsWritten
+                                lineType: LINE
+                    IO Throughput:
+                        units: io in progress
+                        miny: 0
+                        maxy: 1000
```

Comment:
`maxy` shouldn't be set on io in progress. It's likely to be a very small number, but it could go unexpectedly high if there's a lot of disk contention.